<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabDasm">
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
        </font>
       </property>
       <attribute name="title">
        <string>Disassembler</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tvDasm">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideNone</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCOGs">
       <attribute name="title">
        <string>COGs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="saCOGs">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="sawcCOGs">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>662</width>
             <height>374</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="2" column="2">
             <widget class="P2CogView" name="cog5" native="true"/>
            </item>
            <item row="4" column="1">
             <widget class="P2CogView" name="cog8" native="true"/>
            </item>
            <item row="0" column="2">
             <widget class="P2CogView" name="cog1" native="true"/>
            </item>
            <item row="4" column="2">
             <widget class="P2CogView" name="cog9" native="true"/>
            </item>
            <item row="5" column="1">
             <widget class="P2CogView" name="cogA" native="true"/>
            </item>
            <item row="1" column="1">
             <widget class="P2CogView" name="cog2" native="true"/>
            </item>
            <item row="1" column="2">
             <widget class="P2CogView" name="cog3" native="true"/>
            </item>
            <item row="5" column="2">
             <widget class="P2CogView" name="cogB" native="true"/>
            </item>
            <item row="7" column="2">
             <widget class="P2CogView" name="cogF" native="true"/>
            </item>
            <item row="7" column="1">
             <widget class="P2CogView" name="cogE" native="true"/>
            </item>
            <item row="3" column="2">
             <widget class="P2CogView" name="cog7" native="true"/>
            </item>
            <item row="6" column="1">
             <widget class="P2CogView" name="cogC" native="true"/>
            </item>
            <item row="0" column="1">
             <widget class="P2CogView" name="cog0" native="true"/>
            </item>
            <item row="6" column="2">
             <widget class="P2CogView" name="cogD" native="true"/>
            </item>
            <item row="2" column="1">
             <widget class="P2CogView" name="cog4" native="true"/>
            </item>
            <item row="3" column="1">
             <widget class="P2CogView" name="cog6" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="separator"/>
    <addaction name="action_AboutQt5"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Go_to_address"/>
    <addaction name="action_Go_to_COG"/>
    <addaction name="action_Go_to_LUT"/>
    <addaction name="action_Go_to_ROM"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Opcodes_binary"/>
    <addaction name="action_Opcodes_hexdec"/>
    <addaction name="action_Opcodes_octal"/>
    <addaction name="separator"/>
    <addaction name="action_setLowercase"/>
   </widget>
   <widget class="QMenu" name="menu_Run">
    <property name="title">
     <string>&amp;Run</string>
    </property>
    <addaction name="action_SingleStep"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Run"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About …</string>
   </property>
  </action>
  <action name="action_AboutQt5">
   <property name="text">
    <string>About &amp;Qt5</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open …</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Go_to_address">
   <property name="icon">
    <iconset resource="p2emu.qrc">
     <normaloff>:/icons/go_to.png</normaloff>:/icons/go_to.png</iconset>
   </property>
   <property name="text">
    <string>Go to &amp;address …</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Go_to_COG">
   <property name="icon">
    <iconset resource="p2emu.qrc">
     <normaloff>:/icons/go_to_cog.png</normaloff>:/icons/go_to_cog.png</iconset>
   </property>
   <property name="text">
    <string>Go to &amp;COG</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Go_to_LUT">
   <property name="icon">
    <iconset resource="p2emu.qrc">
     <normaloff>:/icons/go_to_lut.png</normaloff>:/icons/go_to_lut.png</iconset>
   </property>
   <property name="text">
    <string>Go to &amp;LUT</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_Go_to_ROM">
   <property name="icon">
    <iconset resource="p2emu.qrc">
     <normaloff>:/icons/go_to_rom.png</normaloff>:/icons/go_to_rom.png</iconset>
   </property>
   <property name="text">
    <string>Go to &amp;ROM</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_setLowercase">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="p2emu.qrc">
     <normaloff>:/icons/lowercase.png</normaloff>:/icons/lowercase.png</iconset>
   </property>
   <property name="text">
    <string>Disassembler &amp;lowercase</string>
   </property>
  </action>
  <action name="action_Opcodes_binary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="p2emu.qrc">
     <normaloff>:/icons/opcode_binary.png</normaloff>:/icons/opcode_binary.png</iconset>
   </property>
   <property name="text">
    <string>Opcodes as &amp;binary</string>
   </property>
  </action>
  <action name="action_Opcodes_hexdec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="p2emu.qrc">
     <normaloff>:/icons/opcode_hexdec.png</normaloff>:/icons/opcode_hexdec.png</iconset>
   </property>
   <property name="text">
    <string>Opcodes as &amp;hexadecimal</string>
   </property>
  </action>
  <action name="action_Opcodes_octal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="p2emu.qrc">
     <normaloff>:/icons/opcode_octal.png</normaloff>:/icons/opcode_octal.png</iconset>
   </property>
   <property name="text">
    <string>Opcodes as &amp;octal</string>
   </property>
  </action>
  <action name="action_SingleStep">
   <property name="icon">
    <iconset resource="p2emu.qrc">
     <normaloff>:/icons/single_step.png</normaloff>:/icons/single_step.png</iconset>
   </property>
   <property name="text">
    <string>Single &amp;step</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>P2CogView</class>
   <extends>QWidget</extends>
   <header location="global">p2cogview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="p2emu.qrc"/>
 </resources>
 <connections/>
</ui>
